### ISSUE TITLE
Windows Compatibility

### ISSUE THREAD

**Issue by ninedaleys** [dend/foggycam#17]
Have you developed a Windows compatible version?

**Comment by jjensen4** [comment]
I got it working in Windows using Visual Studio 2017 community.  Here are some links for setting up Python in Visual Studio and cloning a repository: 
https://docs.microsoft.com/en-us/visualstudio/python/installing-python-support-in-visual-studio
https://docs.microsoft.com/en-us/visualstudio/python/quickstart-03-python-in-visual-studio-project-from-repository
Once the repository is cloned and you follow steps 5-9 in the second url to setup a solution; I did the following to get it to run and create my videos:
1. Go to Python Environments and install azure-storage and azure-storage-blob
2. In your repository source folder create a folder called "tools" and add ffmpeg.exe to the folder - I downloaded ffmpeg version 4.0.2 windows 32 bit from the following site:  https://ffmpeg.zeranoe.com/builds/ 
3. In the foggycam.py script add ".exe" to the file path or it will not recognize ffmpeg as a file:
`ffmpeg_path = os.path.abspath(os.path.join(os.path.dirname( __file__ ), '..', 'tools', 'ffmpeg`**.exe**`')) `
